glib/src/object.rs

```diff
@@ -1372,6 +1372,13 @@ pub trait ObjectExt: ObjectType {
         f: F,
     ) -> SignalHandlerId;
     fn notify<'a, N: Into<&'a str>>(&self, property_name: N);
+
+    #[doc(alias = "g_object_freeze_notify")]
+    fn freeze_notify(&self);
```

Comment:
It would be better to return a guard object here that `thaw`s on `Drop`. Otherwise it's easy to forget to do that in all return paths.